client/src/components/AboutBanner/AboutBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import "./AboutBanner.css"
+import { Jumbotron, Button, Image } from 'react-bootstrap/lib';
+import { Container, Row } from "../Grid"
+
+
+class AboutBanner extends React.Component {
+
+    render() {
+        return (
+            <Jumbotron xs={12} md={8}>
+                <Container>
+                    <Row>
+                        <h1>Connect with other brewers.</h1>
+                    </Row>
+                    <Row>
+                        <Image xs={6} md={6} lg={6} src='https://images.unsplash.com/photo-1515303900226-036e8b860207?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=83b7b7ecf6d99a4b2742f825fe0d2cf4&auto=format&fit=crop&w=600&q=60' alt='glass of beer' rounded responsive />
+                        <ul xs={6} md={6} lg={6}>
+                            <li><h3>Learn how to brew from others</h3></li>
+                            <li><h3>Share your recipes and get feedback</h3></li>
+                            <li><h3>Connect with similar minded people</h3></li>
+                            <li><h3>Save your own recipes and collect your favorites all in one place</h3></li>
```

Comment:
When using the <h...>, use the number as a guide to how many times you're allowed to use it on the same screen. Here you are using <h3> which means it should only be used 3 times. For this specifically, you should consider styling <li> items in your css file. Remember, use classes that relate to your page, for example, you called this the about page so use:
`className='aboutList'`
or something similar so we can see css for specific components/pages at a glance.